host/include/uhd/rfnoc/radio_ctrl.hpp

```diff
@@ -219,6 +219,20 @@ class UHD_RFNOC_API radio_ctrl :
      */
     virtual std::vector<std::string> get_gpio_banks() const = 0;
 
+    /*! Set the radio RX port->channel mapping
+     *
+     * \param port
+     * \param chan RX radio channel to associate with specified port
+     */
+    virtual void set_rx_port_chan_map(const size_t port, const size_t chan) = 0;
+
+    /*! Set the radio TX port->channel mapping
+     *
+     * \param port
+     * \param chan TX radio channel to associate with specified port
+     */
+    virtual void set_tx_port_chan_map(const size_t port, const size_t chan) = 0;
```

Comment:
One minor suggestion, if we have a setter, we should have a getter as well. Using internal private property : _rx_port_to_chan_map is ok but harder to read.

Reply:
Ah, makes sense! The getter would obviously be helpful to expose to outside blocks... right now as a private variable it cannot be read otherwise.